db/views/analytics_exits_v02.sql

```diff
@@ -0,0 +1,66 @@
+SELECT
+  "ExitID",
+  "EnrollmentID",
+  "PersonalID",
+  "ExitDate",
+  "Destination",
+  "OtherDestination",
+  "AssessmentDisposition",
+  "OtherDisposition",
+  "HousingAssessment",
+  "SubsidyInformation",
+  "ConnectionWithSOAR",
+  "WrittenAftercarePlan",
+  "AssistanceMainstreamBenefits",
+  "PermanentHousingPlacement",
+  "TemporaryShelterPlacement",
+  "ExitCounseling",
+  "FurtherFollowUpServices",
+  "ScheduledFollowUpContacts",
+  "ResourcePackage",
+  "OtherAftercarePlanOrAction",
```

Comment:
Suggest removing all of these, they are not in the current version of the HUD Spec, which leads me to believe they are likely from old versions of the spec. (I did spot-check that for some but not all).

```suggestion
```